adaboost/cuda/core/operations.hpp

```diff
@@ -34,8 +34,7 @@ namespace adaboost
             */
 
             template <class data_type_matrix>
-            void fill(const data_type_matrix value, const MatrixGPU<data_type_matrix>&mat, unsigned block_size_x, unsigned block_size_y);
```

Comment:
Why is this deleted?

Reply:
Since in current function, we are only taking one parameter, it number of streams. Instead of taking block_size_x and block_size_y. So the API is different now.

Reply:
Please do not deprecate APIs until utmost necessary. Please keep both APIs, we will use whatever required while implementing the AdaBoost algorithm.
Just copy the original `fill` and related tests function from `master` branch and add it to the right places to avoid conflicts. Do not modify the changes in your patch, just pick the right code from `master` and paste it in your branch.

Reply:
Done, please have a look